### DISCUSSION TITLE
How I can delete my account?/

### DISCUSSION THREAD

**Discussion by ramenfing** [wger-project/wger#1789] [General]
Hello, good morning, 

How I can delete my account? I'm trying to send an email to hello@wger.de but the email address doesn't exist anymore. 

Maybe you should update this in your terms of service website:

https://wger.de/es/software/terms-of-service

Thank you, regards.

**Comment by rolandgeider** [comment]
hi! you can delte your account in preferences / options / delete account

**Reply by JJJul1en** [reply]
Hello, I checked and couldn't find a delete account option. I'm on Android using wger version 1.7.9 downloaded from the Google app store.

**Reply by rolandgeider** [reply]
hey! At the moment this is only possible over the web interface
